credentials_manager/lib/credentials_manager.rb

```diff
@@ -1,3 +1,4 @@
+require 'fastlane_core'
```

Comment:
_credentials_manager_ doesn't have a dependency to _fastlane_core_. Let's not use the new env manager in _credentials_manager_ for now 👍